app/src/main/java/com/x8bit/bitwarden/data/autofill/fido2/datasource/network/model/PasskeyAttestationOptions.kt

```diff
@@ -7,7 +7,7 @@ import kotlinx.serialization.Serializable
  * Models a FIDO 2 credential creation request options received from a Relying Party (RP).
  */
 @Serializable
-data class PublicKeyCredentialCreationOptions(
+data class PasskeyAttestationOptions(
```

Comment:
I'm going to rename this in a separate PR and move it to another package, `data/autofill/fido2/model`. It should have been there originally because it is not a network request model. We'll want `PublicKeyCredentialDescriptor` in that package too.

Reply:
#3584